docker/wordpress_xdebug/Dockerfile

```diff
@@ -6,4 +6,7 @@ RUN pecl install xdebug \
 	&& echo 'xdebug.remote_autostart=1' >> $PHP_INI_DIR/php.ini \
 	&& docker-php-ext-enable xdebug
 RUN apt-get update \
-	&& apt-get install --assume-yes --quiet --no-install-recommends gnupg2 subversion mariadb-client
+	&& apt-get install --assume-yes --quiet --no-install-recommends gnupg2 subversion mariadb-client less
+RUN curl -O https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar \
+	&& chmod +x wp-cli.phar \
+	&& mv wp-cli.phar /usr/local/bin/wp
```

Comment:
There is an alternative method of setting up the cli using `wordpress:cli` container (see below). I'm not sure if it's better but it feels safer then downloading GH builds. What do you think?

```Dockerfile
# At the top of the file
FROM wordpress:cli AS cli
...
# After RUN command
COPY --from=cli /usr/local/bin/wp /usr/local/bin/wp
```

Reply:
We can update that again if there's a need. This is more for daily usage.